src/azul/plugins/metadata/hca/__init__.py

```diff
@@ -503,10 +505,10 @@ def from_metadata(cls,
         parameter_suffix = '; dcp-type=data'
         if content_type.endswith(parameter_suffix):
             content_type = content_type.removesuffix(parameter_suffix)
+        elif config.catalogs[catalog].atlas == 'lungmap':
```

Comment:
```suggestion
        # FIXME: Re-enable content-type validation for all atlases
        #        https://github.com/DataBiosphere/azul/issues/7244
        elif config.catalogs[catalog].atlas == 'lungmap':
```

https://github.com/DataBiosphere/azul/issues/7244#issuecomment-3400087272